test/test_individual.py

```diff
@@ -294,3 +295,17 @@ def test_update_parameters_from_torch_class_does_not_reset_fitness_for_unused_pa
     g = _unpack_evaluation(individual.to_func(), individual_type)
     x = 2.0
     assert g([x])[0] == pytest.approx(x ** 2)
+
+
+@pytest.mark.parametrize("individual_type", ["SingleGenome", "MultiGenome"])
+def test_individual_randomize_genome(individual_type, rng_seed):
+    rng = np.random.RandomState(rng_seed)
+    primitives = (cgp.Add, cgp.Mul)
+    genome = cgp.Genome(1, 1, 2, 1, 1, primitives)
+    genome.randomize(rng)
+
+    dna_old = copy.copy(genome.dna)
```

Comment:
i think `list(genome.dna)` would also work and avoid the import.

Reply:
(bump)

Reply:
Ah thanks, I had forgotten about that. Fixed now.